tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Microsoft.Health.Extensions.BuildTimeCodeGenerator.targets

```diff
@@ -3,9 +3,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
-    <ToolTargetFramework>net5.0</ToolTargetFramework>
-    <ToolTargetFramework Condition="'$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'net5.0'">$(TargetFramework)</ToolTargetFramework>
-    <GeneratorPath>$(MSBuildThisFileDirectory)../../tools/$(ToolTargetFramework)/Microsoft.Health.Extensions.BuildTimeCodeGenerator.dll</GeneratorPath>
+    <GeneratorPath>$(MSBuildThisFileDirectory)../../tools/net6.0/Microsoft.Health.Extensions.BuildTimeCodeGenerator.dll</GeneratorPath>
   </PropertyGroup>
```

Comment:
>net6 [](http://example.com/codeflow?start=58&length=4)

I think this would make generator to be unusable for anything other than .net6

Reply:
It will still work with `net5.0`, as the `ToolTargetFramework` will update for either `net5.0` or `net6.0`. However, in the future if we wanted to use this tool with a `net7.0` assembly, the target file would choose the `net6.0` assembly to use.

Reply:
maybe I'm using generator in a wrong way...

Error MSB3073 The command "dotnet "C:\Users\ivmatan\.nuget\packages\microsoft.health.extensions.buildtimecodegenerator\3.2.38\build\netstandard2.0\../../tools/net6.0/Microsoft.Health.Extensions.BuildTimeCodeGenerator.dll" --generator-name "ImmutableSqlModelGenerator" --output-file "C:\src\fhir-server\src\Microsoft.Health.Fhir.SqlServer\Features\Schema\Model\ImmutableTypes.Generated.cs" --namespace "Microsoft.Health.Fhir.SqlServer.Features.Schema.Model" " exited with code 1.